docs/lab/overview/insurance.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Insurance Ordering"
+---
+
+Vital supports ordering with commercial insurance billing. To set this up, there are a set of steps required. This document explains the complete flow for ordering via commercial insurance.
```

Comment:
I would add clarifying. Note that only commercial payors are supported and public insurance payors (medicare, medicaid, etc.) are not supported.